run-tests.sh

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env bash
 
+# terminate script on error
+set -e
+
+# echo commands to output
+set -x
```

Comment:
Love the inline documentation for these. I don't know how many times I've looked those `set -e` and `set -x` up (and/or gotten them backwards when I didn't).

Reply:
Exactly, why couldn't those switches be easier to remember? :)